### DISCUSSION TITLE
Is it possible to style "<chakraComponents.Option"?

### DISCUSSION THREAD

**Discussion by dzienisz** [csandman/chakra-react-select#320] [Q&A] [answered]
I would like to style `<chakraComponents.Option` is it possible?

**Comment by csandman** [comment, marked as answer]
Yeah, the easiest way to style it would be to use the `chakraStyles` prop. Even if you're using a custom wrapper for it, the `chakraStyles` will still apply.

```tsx
        <Select
          chakraStyles={{
            option: (base) => ({
              ...base,
              backgroundColor: 'blue.200',
            }),
          }}
        />
```

<img width="589" alt="image" src="https://github.com/csandman/chakra-react-select/assets/9214195/fa1bd518-5503-4384-88e2-2171c4826c67">

Here's a demo: https://stackblitz.com/edit/vitejs-vite-qd93qr?file=src%2Fapp.tsx
